tables/th-g0-backward.utb

```diff
@@ -0,0 +1,222 @@
+# liblouis: Thai Current Braille System.
+#
+#-index-name: Thai
+#-display-name: Thai current braille
+#
+#+language:th
+#+region:th-TH
+#+type:literary
+#+system:th-standard
```

Comment:
In this table system is "th-standard", while in th-g0.utb it is "thai". Shouldn't it be the same for both tables?

Reply:
You can change it to Thai.

Reply:
OK, thanks.

Reply:
Done